### ISSUE TITLE
remote write 2.0 - formal proposal

### ISSUE THREAD

**Issue by alexgreenbank** [prometheus/prometheus#13887]
### Proposal

Part of: https://github.com/prometheus/prometheus/issues/13105

Tasks:
- [ ] Follow the process at https://github.com/prometheus/proposals

**Comment by kushalShukla-web** [comment]
hey @bwplotka did you create pr for this issue , if not i would like to work on it :)

**Comment by bwplotka** [comment]
Thanks! We already started and it's tricky even with a lot of context, plus we still discuss details, so we will take it, but feel free to check other remote write items on the issues!

**Comment by kushalShukla-web** [comment]
After reviewing the Remote Write documentation, I am excited about how it supports stateless implementations and why we opted for Remote Write over gRPC. This aspect of the repository truly captivates my interest, and I'm eager to contribute to this area .

**Comment by bboreham** [comment]
Can this be closed since we merged RW 2.0 ?

**Comment by bwplotka** [comment]
No, there's a formal thing we want to merge. We wanted to have it open until stability for the feedback, but now it's probably time to merge it (https://github.com/prometheus/proposals/pull/35)
